builtin/providers/aws/wafregionl_token_handlers.go

```diff
@@ -0,0 +1,50 @@
+package aws
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/waf"
+	"github.com/aws/aws-sdk-go/service/wafregional"
+	"github.com/hashicorp/errwrap"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+type WafRegionalRetryer struct {
+	Connection *wafregional.WAFRegional
```

Comment:
I thought my original `WafRetryer` would be useful for you here too, but I never realized it's a different service with a different name 🤦‍♂️  

Oh well, good decision to create a copy of it. 👍